app/javascript/components/change-logs/components/change-log-item/component.spec.js

```diff
@@ -0,0 +1,29 @@
+import { mountedComponent, screen } from "test-utils";
```

Comment:
Replace old file instead of replacing new file

Reply:
this one is old file.